cumulusci/core/config/BaseProjectConfig.py

```diff
@@ -525,7 +538,9 @@ def pretty_dependencies(self, dependencies, indent=None):
                 prefix = "{}    ".format(" " * indent)
         return pretty
 
-    def process_github_dependency(self, dependency, indent=None, include_beta=None):
+    def process_github_dependency(  # noqa: C901
```

Comment:
I mean, it really _is_ too complex, isn't it?

Reply:
😞